examples/snuba-generic-metrics/1/snuba-generic-metrics-dist-encoded-plain-array.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": 2,
+  "use_case_id": "spans",
+  "org_id": 1,
+  "project_id": 3,
+  "metric_id": 65563,
+  "timestamp": 1704614940,
+  "sentry_received_timestamp": 1704614940,
+  "tags": {
+    "9223372036854776010": "production",
+    "9223372036854776017": "healthy",
+    "65690": "metric_e2e_spans_dist_v_VUW93LMS"
+  },
+  "retention_days": 90,
+  "mapping_meta": {
+    "d": { "65560": "d:spans/duration@second" },
+    "h": {
+      "9223372036854776017": "session.status",
+      "9223372036854776010": "environment"
+    },
+    "f": { "65691": "metric_e2e_spans_dist_k_VUW93LMS" }
+  },
+  "type": "d",
+  "value": { "format": "array", "data": [0.0, 1.1, 2.2, 3.3, 4.4, 5.5] }
```

Comment:
@Dav1dde Maybe let's just double check that this matches with what we expect Relay to output

Reply:
That's what Relay sends:

https://github.com/getsentry/relay/blob/bacbfbede5f1d57140968c202f1c544b9c1d35ce/tests/integration/test_metrics.py#L1877-L1879